include/cassandra.h

```diff
@@ -144,6 +144,11 @@ typedef struct CassUuid_ {
   cass_uint64_t clock_seq_and_node;
 } CassUuid;
 
+#ifdef __cplusplus
```

Comment:
Please remove this block. This is broken without adding it to build and packaging.